composer.json

```diff
@@ -1,5 +1,17 @@
 {
+	"autoload-dev": {
+		"psr-4": {
+			"OCP\\": "vendor/christophwurst/nextcloud/OCP",
+			"OCA\\Talk\\": "lib/"
+		}
+	},
+	"scripts": {
+		"cs:check": "php-cs-fixer fix --dry-run --diff",
+		"cs:fix": "php-cs-fixer fix",
+		"lint": "find . -name \\*.php -not -path './vendor/*' -not -path './build/*' -print0 | xargs -0 -n1 php -l"
+	},
     "require-dev": {
-        "php-parallel-lint/php-parallel-lint": "^1.2"
-    }
+		"nextcloud/coding-standard": "^0.5.0",
+		"christophwurst/nextcloud": "dev-master"
```

Comment:
as that would lock it to a certain commit I would say don't add this here but install only dynamically on CI

Reply:
why would that be locked? Only the JSON is shipped.

Reply:
then scratch what I said. I simply assume the lock file would be in git as well :zipper_mouth_face:

Reply:
yeah let me include this